hosts/icarus/hardware-configuration.nix

```diff
@@ -1,26 +1,29 @@
-# Do not modify this file!  It was generated by ‘nixos-generate-config’
-# and may be overwritten by future invocations.  Please make changes
-# to /etc/nixos/configuration.nix instead.
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs, modulesPath, ... }:
 let
   common = import ../../common/variables.nix;
 in {
   imports =
-    [ <nixpkgs/nixos/modules/installer/scan/not-detected.nix>
+    [ (modulesPath + "/installer/scan/not-detected.nix")
     ];
```

Comment:
Hangin bracket is a lil weird

Reply:
That's how it's generated. Same for all hardware confs.

Reply:
Oh cool